src/resonanceReconstruction/rmatrix/legacy/SpinGroupBase/src/makeReactionIdentifiers.hpp

```diff
@@ -11,11 +11,12 @@ makeReactionIdentifiers( const Channel< Neutron >& channel,
   reactions.push_back( ReactionID{ incident, target, ReactionType( "elastic" ) } );
   reactions.push_back( ReactionID{ incident, target, ReactionType( "capture" ) } );
 
+  // lambdas
+  auto hasFission = [] ( const auto& resonance )
+                       { return resonance.hasFission(); };
+
   // add fission if it exists
-  if ( ranges::count_if(
-           table.resonances(),
-           [&] ( const auto& resonance )
-               { return resonance.hasFission(); } ) > 1 ) {
+  if ( ranges::count_if( table.resonances(), hasFission ) > 1 ) {
```

Comment:
This would work without the `> 1`, but I can't decide if that's better or worse.

Reply:
This is actually wrong: it should be > 0.

Reply:
I fixed this and verified all other uses of count_if: none are wrong.